examples/ray_ddp_example.py

```diff
@@ -103,6 +102,7 @@ def tune_mnist(data_dir,
         num_samples=num_samples,
         resources_per_trial={
             "cpu": 1,
+            "gpu": int(use_gpu),
```

Comment:
this may be controversial -- if you don't set GPU it doesn't work.

Reply:
Yes another good catch, thanks!